atst/forms/forms.py

```
@@ -0,0 +1,12 @@
+import tornado
+from tornado.gen import Return
+from wtforms_tornado import Form
+
+
+class ValidatedForm(Form):
+
+    @tornado.gen.coroutine
+    def perform_extra_validation(self, *args, **kwargs):
+        """A coroutine that performs any applicable extra validation. Must
+        return True if the form is valid or False otherwise."""
+        raise Return(True)
```

Is there a particular reason you did `raise Return(True)` here instead of just `return True`? I think the `raise` method is only necessary for python2, but that seems like something you would have already tried.

---

I had hoped `return True` would work, but [according to the docs](http://www.tornadoweb.org/en/stable/guide/coroutines.html#native-vs-decorated-coroutines)
> Decorated coroutines always return a Future object

When I had `return True`, I was getting an error attempting to `yield` on the result of the call. As far as I can tell, `raise Return(True)` is the necessary way to do it.

---

Yeah, I thought the point of the `@tornado.gen.coroutine` decorator was that it wrapped your return value in a future automatically. Weird that you were getting an error, though.

PR looks good though. :shipit: